pkg/bundle/manager_test.go

```diff
@@ -223,7 +223,7 @@ func TestUpdate(t *testing.T) {
 		}
 		bm := NewBundleManager(logr.Discard(), discovery, puller)
 
-		if assert.True(t, bm.Update(bundle, false, noBundles)) {
```

Comment:
"leaves state as-is (inactive)" should return false if the bundle was already inactive.